autogen/main/cluster.tf.tmpl

```diff
@@ -326,6 +326,11 @@ resource "google_container_cluster" "primary" {
     }
     {% endif %}
   }
+
```

Comment:
nit: I'd suggest moving the new line into the `if` block, as otherwise it's also adding extra lines to the files which don't include.  Will reduce the number of changed files in this PR.